README.md

```diff
@@ -219,6 +219,24 @@ Without further do, it looks like the following:
 
 Here, `matchingRules` represent the assertions of the expectation, while `body`, `query` and `path` contains it's example values.
 
+## Migrating pact-msw-adapter from v2 to v3
```

Comment:
This is a great section.

It might be nice to have a table of compat, wdyt? That way users could see at a glance which package for which version of node and msw they are using

| pact msw version | msw version | node version |

and/or a link at the top of the readme, stating upgrading v2-v3? See here (and link to this section)

Reply:
good idea I'll add that